core/schema/src/main/java/org/hawkular/metrics/schema/SchemaService.java

```diff
@@ -66,51 +65,58 @@ public void run(Session session, String keyspace, boolean resetDB, int replicati
         URI script = getScript();
         cassalog.execute(script, tags, vars);
 
-        session.execute("INSERT INTO " + keyspace + ".sys_config (config_id, name, value) VALUES " +
-                "('org.hawkular.metrics', 'version', '" + getNewHawkularMetricsVersion() + "')");
+        if (updateVersion) {
+            try {
+                updateVersion(session, keyspace, 0, 0);
+            } catch (Exception e) {
```

Comment:
InterruptedException is transformed to VersionUpdateException? I'm slightly confused with these Exception transformations through something that catches RuntimeExceptions also

Reply:
I will clean up the exception handling.